runtime/src/bank_forks.rs

```diff
@@ -503,6 +503,21 @@ impl BankForks {
         )
     }
 
+    /// Set the drop-callback for all the banks, and then enable the bank-drop-callback flag.
+    ///
+    /// This fn shall be called only once, and before Replay starts, IFF AccountsBackgroundService
+    /// shall be responsible for calling AccountsDb::purge_slot() to clean up dropped banks.
+    pub fn set_bank_drop_callback(&mut self, bank_drop_callback: SendDroppedBankCallback) {
```

Comment:
Here's the impl for setting the bank drop callbacks. Note the `&mut self` that I mentioned in the previous comment. This fn doesn't need to be on BankForks, but I found that the fn is the most comfortable and obvious here.